inception-kb/src/main/resources/de/tudarmstadt/ukp/inception/kb/model/db-changelog.xml

```diff
@@ -331,4 +331,35 @@
       </column>
     </addColumn>
   </changeSet>
+
+  <changeSet author="INCEpTION Team" id="20180503">
+    <preConditions onFail="MARK_RAN">
+      <not>
+        <columnExists tableName="knowledgebase" columnName="descriptionIri"/>
+      </not>
+    </preConditions>
+
+    <addColumn tableName="knowledgebase">
+      <column name="descriptionIri"
+              type="VARCHAR(255)"
+              defaultValue="http://www.w3.org/2000/01/rdf-schema#comment">
+        <constraints nullable="false" />
+      </column>
+    </addColumn>
+  </changeSet>
+
+  <changeSet author="INCEpTION Team" id="20180511">
+    <preConditions onFail="MARK_RAN">
+      <not>
+        <columnExists tableName="knowledgebase" columnName="ftsIri"/>
```

Comment:
Mind that the project importer needs to be changed as well when a new non-null IRI is added because previously exported projects do not contain the IRI and a default IRI needs to be set on import.

Reply:
Where do I find the class that needs to be changed?

Reply:
`de.tudarmstadt.ukp.inception.kb.exporter.KnowledgeBaseExporter` and `de.tudarmstadt.ukp.inception.kb.exporter.KnowledgeBaseExporterTest`